k-distribution/include/builtin/domains.k

```diff
@@ -859,7 +859,7 @@ module K-EQUAL
 
   rule K1:Bool ==Bool K2:Bool => K1 ==K K2
 
-  syntax K ::= "#if" Bool "#then" K "#else" K "#fi"   [function, functional, smt-hook(ite), hook(KEQUAL.ite), poly(0, 2, 3)]
+  syntax {Sort} Sort ::= "#if" Bool "#then" Sort "#else" Sort "#fi"   [function, functional, smt-hook(ite), hook(KEQUAL.ite)]
```

Comment:
Not much, if at all.  `Sort` will become a sort variable; in some sense you may say it's bringing the syntax closer to KORE.

Reply:
Ah, I see, I was mistaken about the generated Kore. I thought this generated a symbol in `SortK`, but in fact the Kore symbol is already parametric.

Reply:
Yes, it's basically the same symbol in kore you were already getting, but it's more explicitly obvious in the syntax now. Also, users can now name their sort variables instead of them being automatically numbered.